data/client-side-encryption/README.rst

```diff
@@ -1,1076 +1,2347 @@
-============================
-Client Side Encryption Tests
-============================
+======================
```

Comment:
Is this the CSE spec and not the test README?

Reply:
Yep, I am fixing that!